Frameworks/libsidplayfp/update-generated.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
```

Comment:
To be called after submodule update. Fixed generating `.bin` in upstream https://github.com/libsidplayfp/libsidplayfp/pull/9